### DISCUSSION TITLE
Getting weird console errors

### DISCUSSION THREAD

**Discussion by antonyspericorn** [twbs/bootstrap#36682] [General]
We are using bootstrap 5.1.3 for our project. However we are getting these console errors.  
In the current page, we have enabled  a modal by hardcoding the class "show" and "d-block" as we cannot  trigger via JavaScript due to certain criteria. 

![image](https://user-images.githubusercontent.com/91527854/177555385-209de264-326e-42ce-9678-3fe886031450.png)

![image](https://user-images.githubusercontent.com/91527854/177555981-d6b5560f-ffb8-4a6b-a450-bf0ddb1dbad4.png)

**Comment by TravelTimN** [comment]
@antonyspericorn 

If you're just wanting the modal to appear on page load without the use of JavaScript/jQuery/etc., then it's more than just `show` and `d-block` that you need to have on the modal. Make sure you also have `aria-hidden="false"`, since the default is set to `true`.

Also, make sure to remove the default `fade` class from your modal. This is likely causing your console errors.

Example:
```html
<div class="modal show d-block" id="exampleModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="false">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title" id="exampleModalLabel">Modal title</h5>
        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
      </div>
      <div class="modal-body">
        ...
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
        <button type="button" class="btn btn-primary">Save changes</button>
      </div>
    </div>
  </div>
</div>
```
source: [v5.1 #live-demo](https://getbootstrap.com/docs/5.1/components/modal/#live-demo)

Side Note:
If the modal is always open, then consider not using a modal component, but rather a [card](https://getbootstrap.com/docs/5.1/components/card/#header-and-footer) component.
The modal component being always open might have a fixed position on your screen (unless that's your intention).
The card component can use header/footer like a modal.

Example using card component styled like modal:
```html
<div class="card text-center">
  <div class="card-header ps-5 d-flex justify-content-between">
    Card title
    <button type="button" class="btn-close" aria-label="Close"></button>
  </div>
  <div class="card-body">
    <h5 class="card-title">Not a Modal</h5>
    <p class="card-text">Styled like a modal, but actually a <strong>card</strong></p>
  </div>
  <div class="card-footer text-muted text-end">
    <button type="button" class="btn btn-secondary">Do Something</button>
    <button type="button" class="btn btn-primary">Something Else</button>
  </div>
</div>
```
source: [v5.1 card](https://getbootstrap.com/docs/5.1/components/card/#header-and-footer)

screenshot of example:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/41592028/179393693-da890efe-da32-4c5a-bfe2-17b1a1148d2b.png">
